sdk/src/trace/samplers/parent_or_else.cc

```diff
@@ -1,41 +1,41 @@
-#include "opentelemetry/sdk/trace/samplers/parent_or_else.h"
```

Comment:
Why did this entire file change? Line endings?

Reply:
Probably, my local diff tool seems not reporting the whole file diff. I'll revert the unexpected changes.

Reply:
That worked, thanks.

Reply:
@g-easy seems `IGNORE` is defined as macro to some constant somewhere which causes build error in CI (my local build on Windows is fine). I am trying renaming `IGNORE` to `DROP`. If that works, I'll propose the new name to opentelemetry-spec and merge this PR until the new name is accepted there.

Reply:
I think it is totally fine to follow our current naming convention, instead of taking the names directly from the spec.
For example [this is what OpenTelemetry .NET is doing](https://github.com/open-telemetry/opentelemetry-dotnet/blob/7943d9693898a3cd1f464ed9bf46d2a5d17abed0/src/OpenTelemetry/Trace/SamplingDecision.cs#L22) (it has to be updated to reflect the spec change but you get the idea that we can have language idiomatic names).